casanovo/denovo/model.py

```diff
@@ -913,10 +913,18 @@ def _log_history(self) -> None:
                     ("eval/val_aa_precision", "valid_aa_precision"),
                 ]:
                     metric_value = metrics.get(key, np.nan)
-                    if not np.isnan(metric_value):
-                        self.tb_summarywriter.add_scalar(
-                            descr, metric_value, metrics["step"]
-                        )
+                    if metric_value is not None:
```

Comment:
**suggestion:** This seems a bit overkill. Does it also work by doing `new_style=True` in [`set_scalar(...)`](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_scalar)?

Reply:
This works but it seemed to suppress logging output for the val/train loss. Tensorboard seems to work correctly on dev branch.